lib/plugins/entry-files-manifest.js

```diff
@@ -9,26 +9,40 @@
 
 'use strict';
 
-const EntryFilesManifestPlugin = require('../webpack/entry-files-manifest-plugin');
 const PluginPriorities = require('./plugin-priorities');
-const path = require('path');
 const sharedEntryTmpName = require('../utils/sharedEntryTmpName');
-const manifestKeyPrefixHelper = require('../utils/manifest-key-prefix-helper');
+const AssetsPlugin = require('assets-webpack-plugin');
+
+function processOutput(assets) {
+    if (assets.hasOwnProperty(sharedEntryTmpName)) {
+        delete assets[sharedEntryTmpName];
```

Comment:
We now need to delete one other "magic: entry that may exist :) https://github.com/symfony/webpack-encore/pull/409#discussion_r226689309

Reply:
I will get this done.

Reply:
@weaverryan Is the any other files we need to delete from the output?